torch/csrc/jit/passes/onnx/preclude_inlining.cpp

```diff
@@ -0,0 +1,60 @@
+#include <torch/csrc/jit/passes/onnx/preclude_inlining.h>
+#include <torch/csrc/jit/jit_log.h>
+#include <torch/csrc/jit/passes/onnx/helper.h>
+
+namespace torch {
+namespace jit {
+
+const auto namespaceName = std::string("___torch_mangle_3.");
+const auto qualFuncName =
+    std::string("__torch__.torch.nn.functional."); // + namespaceName;
+const auto qualFuncName2 =
```

Comment:
I need to check the namespaceName and the exact qual string we want to match here.

Reply:
cc @eellison